packages/salesforcedx-vscode-core/src/conflict/metadataCacheService.ts

```diff
@@ -140,7 +159,8 @@ export class MetadataCacheService {
     const operation = await components.retrieve({
       usernameOrConnection: connection,
       output: this.cachePath,
-      merge: false
+      merge: false,
+      suppressEvents: true
```

Comment:
This allows the cache to be loaded (file retrieved) even when it has conflicts.  VSCE stores the file in a tmp dir to facilitate the line-by-line conflict view.